src/log.c

```diff
@@ -167,10 +169,83 @@ static struct fi_ops_log ofi_default_log_ops = {
 	.log = ofi_log,
 };
 
+static int ofi_close_logging_fid(struct fid *fid)
+{
+	return 0;
+}
+
+static int ofi_bind_logging_fid(struct fid *fid, struct fid *bfid,
+				uint64_t flags);
```

Comment:
Move implementation to here to remove the need for a forward declaration.

Reply:
There is another cyclic dependency:
ofi_logging_ops -> ofi_bind_logging_fid() -> ofi_logging_import() -> log_fid -> ofi_logging_ops.

I'm not sure how we can break the dependency.